docs/local-dev-setup.md

```diff
@@ -2,11 +2,12 @@
 
 Follow these steps to set up your local development environment for working with the `terraform-ibm-modules` project.
 
-?> **Tip**: For regular maintenance tasks, see [Regular developer tasks](dev-maintenance.md).
+**Tip**: For regular maintenance tasks, see [Regular developer tasks](dev-maintenance.md).
```

Comment:
this needs to return to the previous syntax: `?>`. It creates the callout/admonition style.